contents/blog/clickhouse-vs-elasticsearch.md

```diff
@@ -0,0 +1,228 @@
+
+Elasticsearch, meanwhile, can accomplish similar performance over certain queries. For instance, if a product needs to print the number of college alumni that are unemployed, Elasticsearch can accomplish this by adding up indices in the inverted index of words that pattern-match to unemployed alumni. In other words, fast search sometimes equates to great analytics by just adding a COUNT() function. 
+
+While both Elasticsearch and ClickHouse are fundamentally backend products, we can compare their respective GUI products – Kibana for Elasticsearch and ClickHouse Cloud for ClickHouse. ClickHouse Cloud is a *much* younger product; Kibana, conversely, has been around for nearly a decade and has an extensive UI.
+
+![Comparinson.png](../images/blog/clickhouse-vs-elastic/comparison.png)
+
+In a nutshell, comparing the analytics efficiency of ClickHouse and Elasticsearch has the same sort-of, not-really awkwardness of other comparisons – they both excel in their respective categories using radically different methods, just catering to a different type of need. However, Elastic’s Kibana product is more mature than ClickHouse Cloud’s competitive offering. 
+
+## Comparing developers
+
+One of the reasons that Elasticsearch grew quickly thanks to fast-paced development by its parent company, Elastic. Elastic has spearheaded the development of Elasticsearch, Kibana, and other accessory products like Beats, a data shipper. For many enterprise customers, Elastic being a psuedo-closed-source solution is balanced by the fact that it can leverage its enterprise revenue to foster a massive engineering effort to improve Elastic.
+
+Conversely, the teams behind ClickHouse are a lot smaller. However, ClickHouse has a very avid developer community, with contributors existing outside of the two major ClickHouse developers – ClickHouse Inc and Altinity Inc. And one of the reasons that ClickHouse is starting to really grow in the last few years is because of its open-source, pro-community brand. 
+
+## Final thoughts
+
+ClickHouse and Elasticsearch are both fantastic solutions for data aggregation and fast search respectively. 
+
+Elasticsearch grew quickly thanks to fast-paced development by its parent company, Elastic. Elastic has spearheaded the development of Elasticsearch, Kibana, and other accessory products like Beats, a data shipper. For many enterprise customers, Elastic being a psuedo-closed-source solution is balanced by the fact that it can leverage its enterprise revenue to foster a massive engineering effort to improve Elastic.
+
+Conversely, while the team behind ClickHouse is smaller, ClickHouse has an avid developer community, with contributors existing outside of the two major ClickHouse developers – ClickHouse Inc and Altinity Inc. And one of the reasons that ClickHouse is starting to grow in the last few years is because of its open-source, pro-community brand, and it's blistering performance.
```

Comment:
These are both repeats of what you said 2 paragraphs ago, probably better to just remove.